lib/logzio-nodejs.js

```diff
@@ -113,6 +114,11 @@ LogzioLogger.prototype.log = function(msg) {
     msg = _.assign(msg, this.extraFields);
     msg.type = this.type;
 
+    if (this.addNanoSecs) {
+        var time = process.hrtime();
+        msg['@timestamp_nano_secs'] = time[1];
```

Comment:
As far as I know kibana doesn't support secondary search fields and I think time[1] doesn't include the main time so I think you'll need to concatenate the two fields as @asafm suggested.